src/rpc/blockchain.cpp

```diff
@@ -2470,6 +2470,60 @@ static RPCHelpMan dumptxoutset()
     };
 }
 
+static RPCHelpMan getblocklocations()
+{
+    return RPCHelpMan{"getblocklocations",
+                "\nEXPERIMENTAL warning: this call may be removed or changed in future releases.\n"
+                "\nReturns a JSON for the location of 'blockhash' and its previous 'nblocks' block and undo data.\n",
```

Comment:
@romanz do you have any plan for what to do with `electrs` if this is removed in the future?

Reply:
If Electrum protocol will allow using the actual `scriptPubKey` for querying the historical transactions, it would be possible to use `blockfilterindex`-based RPC, such as https://github.com/bitcoin/bitcoin/pull/20664 (as suggested [here](https://github.com/bitcoin/bitcoin/pull/14053#issuecomment-747661798)).
Maybe it can be supported with [bwt](https://github.com/shesek/bwt) if the full node has blockfilters?
CC: @SomberNight @shesek